client/src/pages/ScoreBoardPage.css

```diff
@@ -0,0 +1,28 @@
+/* ScoreBoardPage.css */
+.scoreboard {
+    background-color: #4e00a3;
+    padding: 20px;
+    border-radius: 10px;
+    text-align: center;
+    width: 300px;
+    color: #ffea00;
+  }
+  
+  h2 {
```

Comment:
This will style all h2 elements across the entire app. Please replace h2 tag with a class selector.